vueui/src/components/view/cooking/microwave.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <template v-if="on">
+      <p>{{ cookingMessage }}</p>
+      <vui-progress :value="$root.$data.wtime" :min="start_time" :max="start_time + cook_time"/>
+      <vui-button class="danger danger-control" icon="exclamation-triangle" :params="{abort: 1}">Abort!</vui-button>
+    </template>
+    <template v-else>
+      <h4>Ingredients</h4>
+      <ul>
+        <li v-if="!ingredientsPresent">The microwave is empty!</li>
+        <li v-for="(amt, name) in cookingobjs" :key="name">
+          <vui-button push-state :params="{ eject: name }"> {{ name }} • {{ amt }}
```

Comment:
Why am I still seeing push state still set after I scolded you for using it last time?